packages/tailwindcss-react-aria-components/src/index.js

```diff
@@ -5,37 +5,60 @@ const attributes = {
     ['hover', 'hovered'],
     ['focus', 'focused'],
     'focus-visible',
+    'focus-within',
     'pressed',
     'disabled',
     'drop-target',
     'dragging',
     'empty',
+    'allows-dragging',
     'allows-removing',
     'allows-sorting',
     'placeholder',
     'selected',
     'indeterminate',
-    'readonly',
+    ['read-only', 'readonly'],
```

Comment:
I thought it would be confusing to have both the builtin `read-only:` as well as the RAC `readonly:` so I merged them like we did for `hover` and `focus`